backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetAQESuite.scala

```diff
@@ -330,5 +337,44 @@ class GlutenClickHouseTPCHParquetAQESuite
       }
     )
   }
+
+  test("issue-3049 local properties level controller statement fallback") {
```

Comment:
add the test case in the core module?

Reply:
+1. It would be better to hold such test in common module, as it should work regardless of which backend is used.